plugins/org.polymap.rhei.batik/resources/css/md/md-standard.css

```diff
@@ -2533,7 +2533,7 @@ Combo[BORDER]:disabled {
 
 /* line 25, ../../sass/md/_combo.scss */
 Combo-Button {
-  width: 30px;
+  width: 20px;
```

Comment:
Den Hinweis hatte ich Dir gestern 12:39 Uhr im Slack geschrieben, mit dem Verweis auf den PR. Wenn das 30px breit ist, verschwindet sehr viel vom Text und es ist eine Riesenlücke, zwischen abgeschnittenem Textende und Icon selbst.

steffen [12:39 PM] Star this message
Fertig, ich würde die Breite das Buttons auf 20px reduzieren, auch bei der ComboBox. Das PNG selbst ist auch 20 breit, und so ist mehr Platz für den Inhalt. Ich mach nen PR.